includes/wpmenucart-settings.php

```diff
@@ -655,7 +655,9 @@ public function select_element_callback( $args ) {
 		$disabled = (isset( $args['disabled'] )) ? ' disabled' : '';
 		
 		$html  = sprintf( '<select name="%1$s[%2$s]" id="%1$s[%2$s]"%3$s>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $disabled ) );
-		$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), '' );
+		if ( 'shop_plugin' === $args['id'] ) {
```

Comment:
Would it be better to add this for all settings?

Reply:
@Terminator-Barbapapa I have added the condition because the other options are the Pro ones, that do not have a disabled option. 

@alexmigf just in case, could you confirm this?

Reply:
@YordanSoares I understood that the Menu Cart Pro plugin will deactivate the free Menu Cart plugin, and it uses its own methods. So any change here won't affect that.
All of the other uses of this method are for displaying pro settings, and all of them are disabled, so I think it's okay to keep this condition here.

Also, adding this "None" option for menus would be good. This looks like this right now:
![image](https://github.com/wpovernight/wp-menu-cart/assets/136313810/51f0c3ed-d110-49b8-b2e6-444fe43a7344)

Reply:
> I understood that the Menu Cart Pro plugin will deactivate the free Menu Cart plugin, and it uses its own methods. So any change here won't affect that. All of the other uses of this method are for displaying pro settings, and all of them are disabled, so I think it's okay to keep this condition here.

That is right. However, in the Pro plugin there are not "disabled/none" options for those settings, so I think it is better to display the default options for those settings, like this:

![image](https://github.com/wpovernight/wp-menu-cart/assets/38109855/609c76d6-e0f9-4c4e-a585-c4fbcf487682)
 

> Also, adding this "None" option for menus would be good. This looks like this right now:

That is included in this PR as well, could you please try the latest version of `issue-49-improve-explicit-label-for-dropdown`?

Reply:
All good 👌